packages/builder/src/components/automation/SetupPanel/CronBuilder.svelte

```diff
@@ -2,15 +2,36 @@
   import { Button, Select, Input, Label } from "@budibase/bbui"
   import { onMount, createEventDispatcher } from "svelte"
   import { flags } from "stores/backend"
+  import { helpers } from "@budibase/shared-core"
 
   const dispatch = createEventDispatcher()
 
   export let value
+  let error
+
+  const reboot = "@reboot"
```

Comment:
We should probably have this as a constant in `@budibase/types` so that it can be shared through shared-core with the frontend and backend - there is already a string in the backend for this as well in `packages/server/src/automations/utils.ts`